pkg/globalmanager/controllers/federatedlearning/federatedlearningjob.go

```diff
@@ -402,21 +402,118 @@ func IsJobFinished(j *sednav1.FederatedLearningJob) bool {
 	return false
 }
 
+func (c *Controller) getSecret(namespace, name, ownerStr string) (secret *v1.Secret, err error) {
+	if name != "" {
+		secret, err = c.kubeClient.CoreV1().Secrets(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+		if err != nil {
+			err = fmt.Errorf("failed to get the secret %s for %s: %w",
+				name,
+				ownerStr, err)
+		}
+	}
+
+	return
+}
+
+func (c *Controller) getModel(ctx context.Context, namespace, name string) (model *sednav1.Model, secret *v1.Secret, err error) {
```

Comment:
Not only model is returned in `getModel`. How about rename this to?
```suggestion
func (c *Controller) getModelAndItsSecret(ctx context.Context, namespace, name string) (model *sednav1.Model, secret *v1.Secret, err error) {
```

Reply:
fix it